src/components/TasksSubList/TasksSubList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Typography } from '@material-ui/core';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { colors } from 'styles/theme';
+import TaskCheckbox from '../TaskCheckbox';
+import { TasksSubListProps } from './types';
+
+const useStyles = makeStyles({
+  header: {
+    margin: '1rem 0',
+    color: colors.custom.lightText,
+  },
+});
+
+const TasksSubList: React.FC<TasksSubListProps> = ({
+  tasks,
+  count,
+  title,
+  onChange,
+}) => {
+  const { header } = useStyles();
+
+  return (
+    <Box>
+      <Typography className={header} component='h4' variant='h4'>
```

Comment:
As far as I saw the typography you added with 1.6rem is the same as body1. To make it bold you just can copy  ContactDetails to see an example.

Reply:
Ok, so why not to define one more typography dimension for this type of header :
<img width="172" alt="Screenshot 2019-11-21 at 11 30 51" src="https://user-images.githubusercontent.com/5225448/69330140-ad25ab00-0c52-11ea-9188-967e9c5e2e9e.png">
From what I see - it's the same header as all other just with `font-size: 1.6rem`.

> To make it bold you just can copy ContactDetails to see an example.
That's not the best option in terms of code code of conduct. Much better to have things in one place - like one more typography dimension `h4` and just re-use it for all similar cases, instead of using aditional component `<Box fontWeight={'fontWeightBold'}>` just to set bold text.

Reply:
I agree with you, unfortunately It adds extra html element to make the text bold following the guidelines in material-ui.